Documentation/minio-object-store-crd.md

```diff
@@ -0,0 +1,42 @@
+---
+title: Minio Object Store CRD
+weight: 40
```

Comment:
The weight value influences where this doc will appear in the navigation list on https://rook.io/docs/rook/master/.  `40` is already in use by the Monitoring page, so you'd need a value less than that. I reshuffled the numbers around for the CRD docs in my cockroach PR: https://github.com/rook/rook/pull/1777/files#diff-8e7ece48f5174fab91ed2569b2407846

I think you could use 39 right now and be OK.  After the cockroach PR makes it to master, you could use anything from 36-39.